### PR TITLE
Added tw- prefix for all tailwind classes

### PR DESCRIPTION
**Pull request by Previesam**

### CODE REVIEW

**Comment by elreco** [comment]
@Previesam thank you for your contribution! 

It looks like I have a bug here:

image



Do you have the same?

File: tailwind.config.js [hunk 1]
```diff
@@ -17,19 +13,19 @@ module.exports = {
...
+        85: '0.85'
+      }
+    }
   },
-  plugins: [
-    require('@tailwindcss/forms'),
-  ],
-};
+  prefix: 'tw-',
+  important: true,
```
Comment (elreco) [inline tailwind.config.js:29]: Do you really need this?

**Comment by Previesam** [comment]
Yes I do. I am not sure what is wrong. but it worked without issue when
I ran the test. I am checking it.

**Comment by Previesam** [comment]
Yes it is in the development context. but normally it won't affect the
packaged version as that will depend on your project's tailwind config.
Also I just confirmed that the problem is not from the changes I made. The
master branch of the package seems to have the same problem.

**Comment by Previesam** [comment]
I have actually been able to resolve all the problems in this pull on the fork I have in my GitHub account. If you will create a tailwind-prefix branch here I can make a pull request, there instead. By the way Thank you for this awesome project.

File: .gitignore [hunk 2]
```diff
@@ -9,7 +9,7 @@ lerna-debug.log*
 
 node_modules
 .DS_Store
-dist
+#dist
```
Comment (elreco) [inline .gitignore:12]: Why did you remove dist from gitignore here? :o

**Comment by elreco** [comment]
@Previesam thank you my friend!

**Comment by SDIjeremy** [comment]
What if we're not using the tailwind prefix in our own project?

File: .gitignore (hunk 2 above)
Comment (Previesam) [reply .gitignore:12]: Because I need to install using npm directly from github.

**Comment by Previesam** [comment]
> What if we're not using the tailwind prefix in our own project?

You install the original package which does not require prefix.

File: .gitignore (hunk 2 above)
Comment (elreco) [reply .gitignore:12]: Ok I understand

**Comment by jjjrmy** [comment]
Okay, I actually need something like this, but to have a custom prefix. Is that possible?

**Comment by dylanbromby** [comment]
Is this a feature of the main branch? If so, how is it configured? Not seeing the `tailwind-prefix` branch referenced above. We use Tailwind in a project with another CSS library so all our classes are prefixed with `tw-`. Thanks!
